### DISCUSSION TITLE
Improving Accuracy

### DISCUSSION THREAD

**Discussion by taeil** [Berkeley-Data/irrigation_detection#2] [General]
@TsungChinHanKen  CLS methods (BYOL, Mocov2) more sophisticated architecture 
@TsungChinHanKen  data augmentation 
1. temporal augmentation. search for recent paper
2. BigEarthNet data does not have repeated data so need to generate from different data sources (such as Google Earth Engine API) 
- [ ]  diversify augmentation 
- [ ] vision transformer (recent paper)

**Comment by tchken** [comment]
1. temporal augmentation -   https://arxiv.org/pdf/1807.03748.pdf
2. temporal augmentation  -  https://arxiv.org/pdf/2008.03800.pdf 
2. other augmentation - multiview - https://arxiv.org/pdf/1906.05849.pdf
3. transformer - https://papers.nips.cc/paper/2020/file/fa28c6cdf8dd6f41a657c3d7caa5c709-Paper.pdf
4. transformer - https://openai.com/blog/image-gpt/
